### PR TITLE
Change default site url

### PR DESCRIPTION
**Pull request by arturo-seijas**
Add the suffix `.local` to the default site URL. It will now be [application name].local

### CODE REVIEW

**Review by mthaddon — CHANGES_REQUESTED** [review]
Can we update the README as well pls?

**Comment by arturo-seijas** [comment]
> Can we update the README as well pls?

Done

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The charm also supports the `ingress` relation, which can be used with
     juju relate indico:ingress nginx-ingress-integrator:ingress
 
 Once the deployment has completed and the "indico" workload state in
-`juju status` has changed to "active" you can visit `http://indico` in
+`juju status` has changed to "active" you can visit `http://indico.local` in
 a browser (assuming `indico` resolves to the IP(s) of your k8s ingress)
```
Comment (mthaddon) [inline README.md:51]: "assuming `indico.local` resolves"

**Review by mthaddon — COMMENTED** [review]
One other small update needed

File: src/charm.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def _is_configuration_valid(self) -> Tuple[bool, str]:
     def _get_external_hostname(self):
         """Extract and return hostname from site_url."""
         site_url = self.config["site_url"]
-        return urlparse(site_url).hostname if site_url else self.app.name
+        return urlparse(site_url).hostname if site_url else "{}.local".format(self.app.name)
```
Comment (jdkandersson) [inline src/charm.py:125]: Should this be an f-string?

**Review by jdkandersson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (arturo-seijas) [reply README.md]: Thanks

File: src/charm.py (hunk 2 above)
Comment (arturo-seijas) [reply src/charm.py]: Yes

File: src/charm.py [hunk 3]
```diff
@@ -425,9 +425,7 @@ def _on_config_changed(self, event):
             return
         if not self._is_saml_target_url_valid():
             self.unit.status = BlockedStatus(
-                "Invalid saml_target_url option provided. Only {} is available.".format(
-                    UBUNTU_SAML_URL
-                )
+                "Invalid saml_target_url option provided. Only {UBUNTU_SAML_URL} is available."
```
Comment (mthaddon) [inline src/charm.py:428]: Does this need an "f" prefix?

**Review by mthaddon — APPROVED** [review]
LGTM, thx

**Review by merkata — APPROVED** [review]
Very nice, approving.

**Pull request merged**